File: 02/019/main.go
package main

import (
	"fmt"
	"strconv"
	"time"
)

func NumberOfSundays() int {
	var sundays = 0
	for year := 1901; year <= 2000; year++ {
		for month := 1; month <= 12; month++ {
			var strMonth = strconv.Itoa(month)
			var strYear = strconv.Itoa(year)
			var theDay, _ = time.Parse("1-2006", strMonth+"-"+strYear)
			if theDay.Weekday() == time.Sunday {
				sundays++
			}
		}
	}
	return sundays
}

func main() {
	fmt.Println(NumberOfSundays())
}

File: 02/019/main.py
from datetime import datetime

def numberOfSundays():
    sundays = 0
    for year in range(1901, 2000 + 1):
        for month in range(1, 12 + 1):
            theDay = datetime(year, month, 1)
            if theDay.weekday() == 6:
                sundays += 1
    return sundays

print(numberOfSundays())
